common-pg/src/main/scala/com/socrata/pg/store/RollupManager.scala

```diff
@@ -338,6 +338,10 @@ class RollupManager(pgu: PGSecondaryUniverse[SoQLType, SoQLValue], copyInfo: Cop
     SoQLAnalyzerHelper.serialize(baos, analysesColumnId)
     SoQLAnalyzerHelper.deserialize(new ByteArrayInputStream(baos.toByteArray))
   }
+
+  def updateAnalyzer2Rollup(table: String, soql: String): Unit = {
+    
```

Comment:
Verifying that this is supposed to be an empty function.

Reply:
This file wasn't actually supposed to be checked in yet at all.